Database/Corrections/sodQuestFixes.lua

```diff
@@ -2737,6 +2737,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425762,
             [questKeys.zoneOrSort] = sortKeys.HUNTER,
         },
+        [90174] = {
+            [questKeys.name] = "Beast Mastery",
+            [questKeys.startedBy] = {{1187},{407918}},
```

Comment:
No need to show the NPC as well. Also the spawns for 407918 need to be cleaned.

```suggestion
            [questKeys.startedBy] = {nil,{407918}},
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5525/commits/44ef8790cfb545675771bb010db8bc744dd35a5a